src/ruby/Assignment29.java

```diff
@@ -0,0 +1,26 @@
+package ruby;
+/*/Find second Max Number/*/
+
+public class Assignment29 {
+
+	void findSecMax(int[] arr) {
+		int num = 0, secMax = 0;
+		for (int i = 0; i < arr.length; i++) {
+			if (num < arr[i]) {
+				num = arr[i];
+			}
```

Comment:
you can handle both in one loop why you need to two loop?

here you need to put else if() condition like below
else if (input[index] > secondMax && input[index] != max)